command-injection/row_042_ruby_vscode-rdbg_extension.ts

```
@@ -0,0 +1,1038 @@
+					await this.injectRubyEnvironment(command, cwd);
+					break;
+				case VersionManager.Rbenv:
+					command = this.makeShellCommand('rbenv exec ruby' + rubyEnvCommand);
+					await this.injectRubyEnvironment(command, cwd);
+					break;
+				case VersionManager.Rvm:
+					command = this.makeShellCommand('rvm-auto-ruby' + rubyEnvCommand);
+					await this.injectRubyEnvironment(command, cwd);
+					break;
+				case VersionManager.Chruby:
+					const rubyVersion = fs.readFileSync(path.join(cwd!, ".ruby-version"), "utf8").trim();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)